scripts/user_retirement/README.rst

```diff
@@ -0,0 +1,100 @@
+User Retirement Scripts
+=======================
+
+`This <https://github.com/openedx/edx-platform/tree/master/scripts/user_retirement>`_ directory contains python scripts which are migrated from the `tubular <https://github.com/openedx/tubular/tree/master/scripts>`_ respository. 
+These scripts are intended to drive the user retirement workflow which involves handling the deactivation or removal of user accounts as part of the platform's management process.
+
+These scripts could be called from any automation/CD framework.
+
+How to run the scripts
+======================
+
+Download the Scripts
+--------------------
+
+To download the scripts, you can perform a partial clone of the edx-platform repository to obtain only the required scripts. The following steps demonstrate how to achieve this. Alternatively, you may choose other utilities or libraries for the partial clone.
+
+.. code-block:: bash
+
+    repo_url=git@github.com:openedx/edx-platform.git
+    branch=master
+    directory=scripts/user_retirement
+
+    git clone --branch $branch --single-branch --depth=1 --filter=tree:0 $repo_url
+    cd edx-platform
+    git sparse-checkout init --cone
+    git sparse-checkout set $directory
+
+Create Python Virtual Environment
+---------------------------------
+
+Create a Python virtual environment using Python 3.8:
+
+.. code-block:: bash
+
+    python3.8 -m venv ../venv
+    source ../venv/bin/activate
+
+Install Pip Packages
+--------------------
+
+Install the required pip packages using the provided requirements file:
+
+.. code-block:: bash
+
+    pip install -r scripts/user_retirement/requirements/base.txt
+
+In-depth Documentation and Configuration Steps
+----------------------------------------------
+
+For in-depth documentation and essential configurations follow these docs
+
+`Documentation <https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/user_retire/index.html>`_
```

Comment:
We should eventually move this documentation so that it lives near this code.  Can you make sure there is a ticket for that work.

Reply:
https://github.com/openedx/edx-platform/issues/34276
done